editor/src/java/com/defold/editor/Start.java

```diff
@@ -151,6 +187,30 @@ public void start(Stage primaryStage) throws Exception {
             logger.error("failed start", t);
             throw t;
         }
+
+        try {
+            if (Taskbar.isTaskbarSupported() && Taskbar.getTaskbar().isSupported(Taskbar.Feature.MENU) && isPathsAvailable()) {
+                Taskbar taskbar = Taskbar.getTaskbar();
+                PopupMenu dockMenu = new PopupMenu();
+                MenuItem dockMenuItem = new MenuItem("New window");
+                dockMenuItem.addActionListener(new ActionListener() {
+                    @Override
+                    public void actionPerformed(ActionEvent e) {
+                        try {
+                            newWindow();
```

Comment:
We shouldn't re-implement the functionality in Java, rather, we should either invoke the handler directly (although I can't find a convenient way to do that? I think is should be done how we do it from `editor.keymap/execute-command`).

Or we could break out the code in the `:open-project` `defhandler` into a Clojure function that we place next to the handler and call the Clojure function from here. Although to be honest I don't see why this code has to be in Java at all. Couldn't we do this from `boot-open-project/load-stage!` where we do all the other Window and Menu setup?